Makefile

```
@@ -112,8 +112,8 @@ $(WORKDIR)/ipk/content/app/cobalt/content/web/adblock: $(BUILD_WEBOS_YOUTUBE_APP
 	fi
 
 	rm -f $(WORKDIR)/ipk/drm.nfz
-	sed -i 's/YouTube/YouTube Cobalt AdBlock/g' $(WORKDIR)/ipk/appinfo.json
-	jq 'del(.fileSystemType)' < $(WORKDIR)/ipk/appinfo.json > $(WORKDIR)/ipk/appinfo2.json
+	sed -i 's/"title":"YouTube"/"title":"YouTube Cobalt AdBlock"/g' $(WORKDIR)/ipk/appinfo.json
+	jq 'del(.fileSystemType) | .+{"supportGIP": true}' < $(WORKDIR)/ipk/appinfo.json > $(WORKDIR)/ipk/appinfo2.json
```

I would prefer to have a Makefile optino to enable this parameter:
```suggestion
	jq 'del(.fileSystemType) $(if $(PACKAGE_AUTOSTART), | .+{"supportGIP": true})' < $(WORKDIR)/ipk/appinfo.json > $(WORKDIR)/ipk/appinfo2.json
```